projects/igniteui-angular/src/lib/grids/filtering/excel-style/grid.excel-style-filtering.component.ts

```diff
@@ -359,6 +349,12 @@ export class IgxGridExcelStyleFilteringComponent implements OnDestroy, AfterView
 
         this._subMenuOverlaySettings.outlet = (this.grid as any).outlet;
 
+        if (this.excelStyleSearch.column) {
```

Comment:
I don't think we need this check since for internal filtering, the column is always set.

Reply:
Removed